helpers/dbload.ts

```diff
@@ -13,6 +13,7 @@ const sequelize = new Sequelize(
   process.env.DB_PW || 'this',
   {
     host: process.env.DB_HOST || 'city',
+    port: ((process.env.DB_PORT as unknown) as number) || 5432,
```

Comment:
I hope this won't break production.

Reply:
I don't think this will break production since `5432` is the default port for a postgresql database. I'm not sure if we have changed our configurations to use a different port.

However, I think that this `port` key should only be added in when used for development. In case the port for the database we use ever changes, this will ensure that nothing breaks.


```
import { Options, Sequelize } from 'sequelize'
// showing you the import so you can see where I got the Options type

const sequelizeOptions: Options = {
  host: process.env.DB_HOST || 'city',
  logging: false,
  dialect: 'postgres',
  pool: {
    max: 5,
    min: 0,
    acquire: 30000,
    idle: 10000
  }
}
if (process.env.MODE === 'DEV') {
  sequelizeOptions.port = ((process.env.DB_PORT as unknown) as number) || 5432
}
const sequelize = new Sequelize(
  process.env.DB_NAME || 'you',
  process.env.DB_USER || 'failed',
  process.env.DB_PW || 'this',
  sequelizeOptions
)
```

Reply:
Better safe than sorry. Done.